webpack/assets/javascripts/react_app/components/AuthSource/Welcome.js

```diff
@@ -15,10 +15,10 @@ export const WelcomeAuthSource = ({ canCreate }) => {
       <FormattedMessage
         id="LDAP-providers"
         defaultMessage={__(
-          'The authentication process currently requires an LDAP provider, such as {FreeIPA}, {OpenLDAP} or {Microsoft}.'
+          'The authentication process currently requires an LDAP provider, such as {Free_IPA}, {OpenLDAP} or {Microsoft}.'
         )}
         values={{
-          FreeIPA: <em>FreeIPA</em>,
+          Free_IPA: <em>{__('FreeIPA')}</em>,
```

Comment:
For those wondering why this is translated: we (ab)use translations for branding. See https://github.com/RedHatSatellite/foreman_theme_satellite/blob/f78f229ea2f44e34a906ef5c1aaee71586baaf83/lib/foreman_theme_satellite/branded_words.rb#L36